jwst/photom/miri_mrs.py

```diff
@@ -23,13 +21,13 @@ def expfunc_bounded(x, a, b, c, x0):
 
     Parameters
     ----------
-    x : int
+    x : float
         Modified Julian Day (EXPMID) of the observation
-    a : numpy float array
+    a : float numpy.ndarray
         Model parameter for time dependence
```

Comment:
I know you didn't write this, but it's a bit silly that all the parameters are given identical descriptions